app/views/volunteers/edit.html.slim

```diff
@@ -2,7 +2,12 @@
   .col-xs-12
     h1= t_title(:edit)
 
-== render 'form'
+.row
+  .col-xs-12
+    = simple_form_for(@volunteer) do |f|
```

Comment:
If I remove the nesting then there is no space between the update and show buttons.